spec/subjob_processor_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'spec_helper'
+
+describe Sidekiq::Superworker::SubjobProcessor do
```

Comment:
This was a missing test that I created a long time ago that I forgot to check in.